### PR TITLE
Added new book resources for python developers

### PR DESCRIPTION
**Pull request by prottayislive**
It's hard to find interview prep resources on Python. These resources cover all interview queries for python developers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,6 +136,9 @@ complete beginners.
 ## Here is 1 from our huge range...
 
 - [Cracking the Coding Interview](https://www.google.com/url?sa=t&source=web&rct=j&url=https://cin.ufpe.br/~fbma/Crack/Cracking%2520the%2520Coding%2520Interview%2520189%2520Programming%2520Questions%2520and%2520Solutions.pdf&ved=2ahUKEwjypc602JnuAhVUuXEKHd5SCnYQFjAAegQIARAB&usg=AOvVaw3vI5dXqJ_ptFWhjqeUJeY_) - This book is about coding interviews.
+- * [Data Structures and Algorithms in Python 1st ed - Goodrich](https://www.amazon.com/Structures-Algorithms-Python-Michael-Goodrich/dp/1118290275/ref=pd_sbs_9/146-7117112-3214425?pd_rd_w=u0eqq&pf_rd_p=43345e03-9e2a-47c0-9b70-a50aa5ecbd5c&pf_rd_r=GH4061FRHKTZNWK7ZJ8Y&pd_rd_r=c91ffc68-6290-4fc9-a165-950ed977867a&pd_rd_wg=hROgA&pd_rd_i=1118290275&psc=1) - This book has excellent resources on Python Data Structures.
```
Comment (AvidCoder101) [inline README.md:139]: @prottayislive Please remove the `-` and then I everything will be fine.

**Review by AvidCoder101 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MRslimcoder) [reply README.md:139]: Also remove *

File: README.md [hunk 2]
```diff
@@ -136,6 +136,9 @@ complete beginners.
...
+* [Elements of the Programming Interview in Python - Aziz](https://www.amazon.com/Elements-Programming-Interviews-Python-Insiders/dp/1537713949/ref=pd_sbs_3/146-7117112-3214425?pd_rd_w=h1Gnk&pf_rd_p=43345e03-9e2a-47c0-9b70-a50aa5ecbd5c&pf_rd_r=3W8ND85QTCQKRJFW53AW&pd_rd_r=244165c1-9543-4cde-ab20-6fedd20ab61a&pd_rd_wg=Mlj5i&pd_rd_i=1537713949&psc=1) - This book is an alternative to Cracking The Coding Interview Book, for Python developers.
```
Comment (MRslimcoder) [inline README.md:140]: remove *

File: README.md [hunk 3]
```diff
@@ -136,6 +136,9 @@ complete beginners.
...
+* [The Algorithm Design Manual - Skiena](https://www.amazon.com/Algorithm-Design-Manual-Steven-Skiena/dp/1849967202/ref=pd_sbs_7/146-7117112-3214425?pd_rd_w=JNweZ&pf_rd_p=43345e03-9e2a-47c0-9b70-a50aa5ecbd5c&pf_rd_r=YS938S6HF8HDZ3C3VW7T&pd_rd_r=0880b776-764d-4269-88b0-21914e3be232&pd_rd_wg=eUOYw&pd_rd_i=1849967202&psc=1) - One of the most famous algorithm books suggested by CS tutors
```
Comment (MRslimcoder) [inline README.md:141]: remove *

File: README.md [hunk 4]
```diff
@@ -136,6 +136,9 @@ complete beginners.
 ## Here is 1 from our huge range...
```
Comment (MRslimcoder) [inline README.md:136]: replace 3 to 1

**Comment by MRslimcoder** [comment]
@AvidCoder101 @prottayislive  I fixed changes by myself.

**Comment by MRslimcoder** [comment]
@CodingSpecies can you please review this PR as well as.

**Review by AvidCoder101 — APPROVED** [review]
NIce @MRslimcoder

**Comment by MRslimcoder** [comment]
@all-contributors add @prottayislive for content.

**Pull request merged**